Added EventListActivity that shows nearby events.

multicoredump opened the pull request:
ToDo:
- Handle errors such as GPS not enabled
- No events found etc
- Display Event image

+ Also removed MainActivity - was not really being used.

Commit:
Added EventListActivity that shows nearby events.

ToDo:
- Handle errors such as GPS not enabled
- No events found etc

+ Also removed MainActivity - was not really being used.

modiaditya commented on app/src/main/AndroidManifest.xml:
```diff
@@ -1,26 +1,33 @@
-        <activity android:name="com.icebreakers.nexxus.activities.SplashActivity">
+        android:theme="@style/AppTheme.NoActionBar">
+
+        <activity android:name=".activities.SplashActivity">
```
Splash should still be the launch activity

modiaditya commented on app/src/main/res/values/styles.xml:
```diff
@@ -1,11 +1,56 @@
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
+    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
```
We should stick with NoActionBar and use Toolbars instead

modiaditya commented on app/src/main/res/layout/item_event.xml:
```diff
@@ -0,0 +1,38 @@
+    card_view:cardUseCompatPadding="true"
+    card_view:cardCornerRadius="8dp"
+    card_view:cardElevation="3dp"
+    android:background="#F9F8F8"
```
Lets define this color in color.xml

modiaditya commented on app/src/main/res/layout/activity_event_list.xml:
```diff
@@ -0,0 +1,43 @@
+
+    </android.support.design.widget.AppBarLayout>
+
+    <android.support.v4.widget.SwipeRefreshLayout
```
This should be part of the fragement layout

modiaditya commented on app/src/main/java/com/icebreakers/nexxus/activities/EventListActivity.java:
```diff
@@ -0,0 +1,174 @@
+import rx.schedulers.Schedulers;
+import rx.subscriptions.CompositeSubscription;
+
+public class EventListActivity extends AppCompatActivity
```
As we spoke the other day, can you move the core logic to a fragment?

modiaditya reviewed (commented):
Awesome job!

Commit:
Added Group Photo + Event Details Activity

{KNOWN_BUG}
 -  Hook up back button on EventDetailsActivity
 - OnResume() images are displayed shorter inside EventListActivity

Next steps:
Better layout for EventDetailsActivity
Show attendees list - using a fragment
